dask_jobqueue/jobqueue.yaml

```diff
@@ -11,7 +11,8 @@ jobqueue:
     death-timeout: 60           # Number of seconds to wait if a worker can not find a scheduler
     local-directory: null       # Location of fast local storage like /scratch or $TMPDIR
     shared-temp-directory: null       # Shared directory currently used to dump temporary security objects for workers
-    extra: []
+    extra: null
```

Comment:
Nit: Could you add a descriptive comment to each of these like the above config options?

Same for all below.

Reply:
Sure, done!